runtime/freelist.c

```diff
@@ -1780,7 +1780,7 @@ typedef enum caml_policy_t {
   policy_best_fit = 2,
 } caml_policy_t;
 
```

Comment:
Since you're not using the types `caml_policy_t` and `enum caml_policy_t` anywhere else, you could just do
```
enum {
  policy_next_fit = 0,
  policy_first_fit = 1,
  policy_best_fit = 2,
};
```
or, for a more self-documenting style, name the enum but without a typedef:
```
enum caml_policy {
  policy_next_fit = 0,
  policy_first_fit = 1,
  policy_best_fit = 2,
};
```